### ISSUE TITLE
How to integrate ExtReact with CoreUI?

### ISSUE THREAD

**Issue by milosKomiRajkovic94** [sencha/extjs-reactor#403]
Is it even possible to integrate ExtReact with Core UI? Because when I eject CoreUI React project, I can't even do npm run build or npm run start, what is usually working well before eject. And I need to eject in order to follow instructions for integration with ExtReact (https://github.com/sencha/ext-react-cra-eject).

**Comment by mgusmano** [comment]
I have not used CoreUI, but i would think ExtReact can be integrated - does CoreUI use Webpack?

**Comment by milosKomiRajkovic94** [comment]
I have managed to integrate it, first it was mine mistake to say that CoreUI project can't be start or built after npm run eject, it can be, but we must run npm install --save react-scripts once again and then after that I followed instructions for integration with ExtReact with existing create-react-app project. Also, for build  of ExtReact project we can't use  build script: react-scripts-with-sass but node scripts/build.js and then it works fine. Every scss and sass from CoreUI will be read correctly! :) This issue is closed! :)
